test/pom.xml

```diff
@@ -11,6 +11,10 @@
 
     <artifactId>test</artifactId>
 
+    <properties>
+        <maven.deploy.skip>true</maven.deploy.skip>
```

Comment:
We might have to deploy this for now as there are modules which use Corfu's test framework to write their tests.

Reply:
fixed